clusterDeployer.py

```diff
@@ -91,23 +94,24 @@ def setup_vm(h: host.Host, cfg: dict, iso_or_image_path: str):
     OS_VARIANT = "rhel8.6"
     RAM_MB = 32784
     CPU_CORE = 8
-    network = "default"
-
-    cmd = f"""
-    virt-install
-        --connect qemu:///system
-        -n {name}
-        -r {RAM_MB}
-        --cpu host
-        --vcpus {CPU_CORE}
-        --os-variant={OS_VARIANT}
-        --import
-        --network=network:{network},mac={mac}
-        --events on_reboot=restart
-        {cdrom_line}
-        --disk path={cfg["image_path"]}
-        {append}
-    """
+    if h.is_localhost():
+        network = "network=default"
+    else:
+        network = "bridge=virbr0"
+    cmd = f""" virt-install \
+        --connect qemu:///system \
+        -n {name} \
+        -r {RAM_MB} \
+        --cpu host \
+        --vcpus {CPU_CORE} \
+        --os-variant={OS_VARIANT} \
+        --import \
+        --network {network},mac={mac} \
+        --events on_reboot=restart \
+        {cdrom_line} \
+        --disk path={cfg["image_path"]} \
+        {append} """
+
```

Comment:
these lines are added by hand. It seems that we need \ when calling command over ssh. Instead of fixing this here, let's fix it in h.run(cmd) and have that function do `cmd = cmd.replace("\n", "\\\n")` when sending to remote